src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useThorStreamingProgress.tsx

```diff
@@ -83,8 +83,24 @@ export const useThorStreamingProgress = (
         // no payload at all - must be a failed request - return
         if (!updatedStreamingSwapData) return
 
-        // no valid data received so far and no valid data to update - return
-        if (!streamingSwapDataRef.current?.quantity && !updatedStreamingSwapData.quantity) return
```

Comment:
This technically changes runtime behaviour if this condition is ever met during an actual streaming swap, though I don't think that's a state we can get in.

@gomesalexandre you might have some thoughts ser.

Reply:
That conceptually looks good, though, noting with the monkey patch in this diff, we're not getting an *actual* streaming swap memo.

<img width="1585" alt="image" src="https://github.com/user-attachments/assets/223ff678-437f-462b-825d-424ac973915c">

This is weird, because we are indeed getting a streaming memo upstream:

<img width="704" alt="image" src="https://github.com/user-attachments/assets/c77e88c2-8e93-41d2-9a22-e76224034442">
<img width="1449" alt="image" src="https://github.com/user-attachments/assets/19f6101e-7778-47ae-9cfc-1dd3bc221538">

Looks like the streaming params are lost somewhere between `getL1Quote` and execution.

Going down the rabbit hole of swapper flow, I noticed that this does *not* seem to be happening while running many breakpoints, which seems to indicate this may be a race condition, and potentially the reason why we don't *consistently* see streaming swaps as available.

Running this without a debugger and with a *valid* streaming memo, I can indeed confirm there is no *quantity* on a small streaming swap with 10 interval 0 (auto) quantity:

<img width="628" alt="image" src="https://github.com/user-attachments/assets/004f6c74-2595-46a2-b4c3-19bc1ae68574">


Monkey patching the memo to be 1/10/10 instead (note the 1 in base unit as a quantity instead, which will void the upstream auto-limit and may help enforcing a streaming swap), I can confirm there's indeed no quantity upstream:

<img width="625" alt="image" src="https://github.com/user-attachments/assets/ef141910-94ea-4e34-9b41-6f8e6d758b2f">
https://viewblock.io/thorchain/tx/4eb1d8f0006c101268b136a3565e31af88a06ce4647062a33c90d2bfd0d981a8

Reply:
tl;dr @0xApotheosis if this ever happens during an *actual* streaming swap (read: with a quantity intent in the memo), then it means there is no akschual quantity and a regular swap was executed instead, so we should be good to go!